docs/source/architectureref/certificates_path_list_quorum.md

```diff
@@ -0,0 +1,49 @@
+Certificate Paths on Vault for Quorum Network
+---------------------------------------------
+
+### For IBFT
+
+| Path                                                                              | Crypto-material               | Type         |
+|-----------------------------------------------------------------------------------|-------------------------------|--------------|
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | nodekey                       | Public Key   |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | keystore                      | Private Key  |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | db_user                       | Username     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | dn_password                   | Password     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | gethpassword                   | Password     |
+
+
+### For RAFT
+
+| Path                                                                             | Crypto-material               | Type         |
+|----------------------------------------------------------------------------------|-------------------------------|--------------|
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | nodekey                       | Public Key   |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | keystore                      | Private Key  |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | db_user                       | Username     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | dn_password                   | Password     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | gethpassword                   | Password     |
+
+
+### For Tessera
+
+| Path                                                                           | Crypto-material               | Type         |
+|--------------------------------------------------------------------------------|-------------------------------|--------------|
+| sys/policy/vault-crypto-{{ `component_name` }}-ro                              | tm.pub                        | Public Key   |
```

Comment:
The path is wrong here, this seems like a policy.
Add Tessera and Constellation. Add description "Public key of Transaction manager"